lib/buildkite/pipelines/command.rb

```diff
@@ -55,8 +55,14 @@ def initialize(command, subcommand, *args)
       end
 
       def run(capture: false)
-        stdout, _, status = Open3.capture3(*to_a)
-        capture ? stdout : status.success?
+        stdout, stderr, status = Open3.capture3(*to_a)
+        if capture
+          stdout
+        elsif status.success?
+          true
+        else
+          raise stderr
```

Comment:
Depending on where you are catching this, may want to make your own error class that has access to both stdout and stderr.